contracts/Lens/InterestRateModelLens.sol

```diff
@@ -30,7 +30,7 @@ contract InterestRateModelLens {
             uint256 borrowRate = ir.getBorrowRate(cash, borrow, reserves);
             borrowSimulation[percent_Factor - 1] = borrowRate;
 
-            uint256 supplyRate = ir.getSupplyRate(cash, borrow, reserves, reserveFactorMantissa);
+            uint256 supplyRate = ir.utilizationRate(cash, borrow, reserves);
```

Comment:
It seems wrong, the variable is `supplyRate` but we are getting the utilization rate

Reply:
Fixed in [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/5488cd4a76d91c22462155a27df00bf5de2fe8a0)